### DISCUSSION TITLE
特定のキーが反応しない

### DISCUSSION THREAD

**Discussion by naoishii** [Yowkees/keyball#392] [Q&A] [answered]
既出の質問で恐縮ですが、ご助言いただけると助かります。

機種：Keyball39 ver2
症状：使い始めでは正常に入力できていて、１、２週間ほどで反応が悪くなり、また調子を戻したと思ったらまた極端に反応が悪くなりました。接触不良を疑いハンダし直したらキーの入力ができなくなりました。
確認したこと：
- 動作しないキーのPCBソケットにテスターを当てて、通電がないことを確認
- その他の動作するキーのPCBソケットにテスターをあてて、通電があることを確認
- ソケットを外してピンセットでショートさせて、入力ができないことを確認
- 表側からピンセットでショートさせて、入力ができないことを確認

以上、どうぞよろしくお願いいたします。

![PXL_20231019_144830422](https://github.com/Yowkees/keyball/assets/3609176/4ffbd704-de90-40b1-bbf9-3ab583a22bb9)

**Comment by Ryo624** [comment, marked as answer]
ご連絡ありがとうございます。

ご連絡いただいたキーに対応するダイオードは赤丸で示したダイオードになります。
半田が球体になっていますので、テンプラ（接続不良）起こしている可能性が高いです。

該当するダイオードの半田付けの見直しをよろしくお願いいたします。
![276660572-4ffbd704-de90-40b1-bbf9-3ab583a22bb9](https://github.com/Yowkees/keyball/assets/143698349/24ceed27-3f5c-4050-a335-0ce0b93b2a72)

**Reply by naoishii** [reply]
ご回答ありがとうございます！
ご指摘いただいたダイオードをハンダし直したところ、正常に動作いたしました。
迅速なご対応ありがとうございました！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
末永くご愛用していただけますと幸いです。

**Comment by n350071** [comment]
奇遇にも、まったく同じ箇所、まったく同じ原因でした。
この質問と回答に助けられました。
ありがとうございます。
